README.md

```diff
@@ -249,39 +249,43 @@ Usage:
+      --directories-per-second-rate-limit int                The maximum number of directories to crawl per second. 0 for unlimited.
+      --disable-cve-2021-44832-detection                     Disable detection of CVE-2021-44832 in versions up to 2.17.0
+      --disable-cve-2021-45105-detection                     Disable detection of CVE-2021-45105 in versions up to 2.16.0
+      --disable-detailed-findings                            Do not print out detailed finding information when not outputting in JSON.
+      --disable-flagging-jndi-lookup                         Do not report results that only match on the presence of a JndiLookup class.
+                                                             Even when disabled results which match other criteria will still report the presence of JndiLookup if relevant.
+      --disable-unknown-versions                             Only output issues if the version of log4j can be determined (note that this will cause certain detection mechanisms to be skipped)
+      --enable-obfuscation-detection                         Enable applying partial bytecode matching to Jars that appear to be obfuscated. (default true)
+      --enable-partial-matching-on-all-classes               Enable partial bytecode matching to all class files found.
+      --enable-trace-logging                                 Enables trace logging whilst crawling. disable-detailed-findings must be set to false (the default value) for this flag to have an effect.
+      --file-path-only                                       If true, output will consist of only paths to the files in which CVEs are detected
+  -h, --help                                                 help for crawl
+      --ignore-dir strings                                   Specify directory pattern to ignore. Use multiple times to supply multiple patterns.
+                                                             Patterns should be relative to the provided root.
+                                                             e.g. ignore "^/proc" to ignore "/proc" when using a crawl root of "/"
+      --json                                                 If true, output will be in JSON format
+      --maximum-average-obfuscated-class-name-length int     The maximum class name length for a class to be considered obfuscated. (default 3)
+      --maximum-average-obfuscated-package-name-length int   The maximum average package name length a class to be considered obfuscated. (default 3)
+      --nested-archive-disk-swap-dir string                  When nested-archive-disk-swap-max-size is non-zero, this is the directory in which temporary files will be created for writing temporary large nested archives to disk. (default "/tmp")
+      --nested-archive-disk-swap-max-size uint               The maximum size in bytes of disk space allowed to use for inspecting nest archives that are over the nested-archive-max-size.
```

Comment:
We should say what the default is here

Reply:
(And in the actual implementation too of course)

Reply:
Added in https://github.com/palantir/log4j-sniffer/pull/98/commits/75761ecab605535572e6dc89d5009c4c1ab36b46

I couldn't quite find where in the implementation to add it as the default value of the exported struct field is 0, where the current comment is true in that 0 is a valid maximum number of bytes to be used. Let me know if I'm misunderstanding here and you feel that is still valuable to document, though :)

Reply:
Sorry, I meant in the text to document the default being 0 i.e. disabled